src/test/scala/uk/gov/hmrc/test/AkkaMaterializerSpec.scala

```diff
@@ -0,0 +1,38 @@
+/*
```

Comment:
no, AkkaMaterializerSpec is needed by the new test but not any other tests. Try removing `with AkkaMaterializerSpec` from `TestWithoutPlayAppSpec` and then running `TestWithoutPlayAppSpec` - it will fail. Whereas all the other tests run fine without it